lib/iex/lib/iex/helpers.ex

```diff
@@ -393,7 +393,14 @@ defmodule IEx.Helpers do
   end
 
   @doc """
-  Prints information about the given data type.
+  Prints information about the data type of a given term.
+  
+  ## Examples
+
+      i([1,2,3])
+      i("Elixir rocks!")
+      i(:some_atom)
+      i(fn x -> x * 2 end)
```

Comment:
I think we should a least have one example to make clear that you give it some Elixir value and it will give you information about it's type. The term term isn't so friendly on beginners.